framework/src/meshgenerators/XYDelaunayGenerator.C

```diff
@@ -115,7 +115,8 @@ std::unique_ptr<MeshBase>
 XYDelaunayGenerator::generate()
 {
   // Put the boundary mesh in a local pointer
-  std::unique_ptr<UnstructuredMesh> mesh = dynamic_pointer_cast<UnstructuredMesh>(_bdy_ptr);
+  std::unique_ptr<UnstructuredMesh> mesh =
+      dynamic_pointer_cast<UnstructuredMesh>(std::move(_bdy_ptr));
```

Comment:
we didnt use to get local ownership of it. 
I havent gotten to it yet, but this means we need the duplication in the backend

Reply:
we didn't use to get local ownership which meant that once generate goes out of scope, the mesh is still available and not destructed even though it's not needed anymore

Reply:
and duplication in the backend already exists and works

Reply:
ok good. I m not sure where we were on keeping the duplication